test/support/api.ts

```diff
@@ -17,15 +17,27 @@ export class Api {
     return this.request().get(url).set(this.getValidAuthHeader());
   }
 
-  post(url: string, body: string | object): request.Test {
-    return this.request().post(url).send(body).set(this.getValidAuthHeader());
+  post(url: string, body: string | unknown | unknown[], extraHeaders?: Record<string, string>): request.Test {
```

Comment:
Why can't we have `body: string | object`?

Reply:
Abhi wanted type for helper `prepareModifiedRequest` and this caused chain reaction in typescript.

We used `unknown | unknown[]` in other places for request body. We could drop support for array, but helper is from tfs-api, and would be nice to keep them consistent for possibly sharing components later.